From b4a99395fe5ac1de33d85a695c5caff1f34adf5d Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 8 May 2018 11:07:58 +0800
Subject: [PATCH 3749/5242] MLK-18229-3 gpu: imx: dpu: fetchdlayer: Cosmetic
 changes on fl_ops entries

commit  e993ef613c676cf6906ecd0db57bb39f8064a05a from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch contains cosmetic changes on fl_ops entries to make the
entries be more readable.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-fetchlayer.c |   32 ++++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/drivers/gpu/imx/dpu/dpu-fetchlayer.c b/drivers/gpu/imx/dpu/dpu-fetchlayer.c
index 319f3d5..15738a0 100644
--- a/drivers/gpu/imx/dpu/dpu-fetchlayer.c
+++ b/drivers/gpu/imx/dpu/dpu-fetchlayer.c
@@ -241,22 +241,22 @@ void dpu_fl_put(struct dpu_fetchunit *fu)
 EXPORT_SYMBOL_GPL(dpu_fl_put);
 
 static const struct dpu_fetchunit_ops fl_ops = {
-	.set_burstlength = fetchunit_set_burstlength,
-	.set_baseaddress = fetchunit_set_baseaddress,
-	.set_src_bpp = fetchunit_set_src_bpp,
-	.set_src_stride = fetchunit_set_src_stride,
-	.set_src_buf_dimensions = fetchlayer_set_src_buf_dimensions,
-	.set_fmt = fetchlayer_set_fmt,
-	.enable_src_buf = fetchunit_enable_src_buf,
-	.disable_src_buf = fetchunit_disable_src_buf,
-	.is_enabled = fetchunit_is_enabled,
-	.set_framedimensions = fetchlayer_set_framedimensions,
-	.set_controltrigger = fetchlayer_set_controltrigger,
-	.get_stream_id = fetchunit_get_stream_id,
-	.set_stream_id = fetchunit_set_stream_id,
-	.pin_off = fetchunit_pin_off,
-	.unpin_off = fetchunit_unpin_off,
-	.is_pinned_off = fetchunit_is_pinned_off,
+	.set_burstlength	= fetchunit_set_burstlength,
+	.set_baseaddress	= fetchunit_set_baseaddress,
+	.set_src_bpp		= fetchunit_set_src_bpp,
+	.set_src_stride		= fetchunit_set_src_stride,
+	.set_src_buf_dimensions	= fetchlayer_set_src_buf_dimensions,
+	.set_fmt		= fetchlayer_set_fmt,
+	.enable_src_buf		= fetchunit_enable_src_buf,
+	.disable_src_buf	= fetchunit_disable_src_buf,
+	.is_enabled		= fetchunit_is_enabled,
+	.set_framedimensions	= fetchlayer_set_framedimensions,
+	.set_controltrigger	= fetchlayer_set_controltrigger,
+	.get_stream_id		= fetchunit_get_stream_id,
+	.set_stream_id		= fetchunit_set_stream_id,
+	.pin_off		= fetchunit_pin_off,
+	.unpin_off		= fetchunit_unpin_off,
+	.is_pinned_off		= fetchunit_is_pinned_off,
 };
 
 void _dpu_fl_init(struct dpu_soc *dpu, unsigned int id)
-- 
1.7.9.5

